spec/openAPI.yml

```
@@ -12,13 +12,13 @@ externalDocs:
   description: Learn more about Funnelback search
   url: https://docs.funnelback.com/
 servers:
-  - url: http://york.funnelback.co.uk/s/
+  - url: http://york.funnelback.co.uk/
```

This seems to work, don't know if it'll fix the network error we get when trying it out too

```suggestion
  - url: https://york.funnelback.co.uk/
```